src/univariate/continuous/loguniform.jl

```diff
@@ -53,8 +53,8 @@ function pdf(d::LogUniform, x::Real)
 end
 function cdf(d::LogUniform, x::Real)
     x1, a, b = promote(x, params(d)...)
```

Comment:
Is there an advantage to use explicit `promote` instead of just doing the computation?

Reply:
Yes, no unintended peomotions for `Int` + `Float32`. It's the same reason as explained in `pdf`.

Reply:
@jw3126 Can you add a similar comment as for `pdf` to all promotions, please?

Reply:
Ah, I see, `1/Int` pollutes with `Float64`

Reply:
"pollute" no offence to the `Int`s 😁

Reply:
done